src/components/VideoPlayer/index.js

```diff
@@ -171,6 +171,78 @@ export default class VideoPlayer extends PureComponent {
 
   resumeVideo = () => { this.video.play() }
 
+  handleKeyDown = (e) => {
+    const { target } = e
+    const selectingVolume = target.className.indexOf('vjs-volume-bar') > -1 ||
+      target.className.indexOf('vjs-volume-menu-button') > -1
+
+    if (selectingVolume) {
+      // trying to prevent these listeners from affecting accessibility controls
+      return
+    }
+
+    switch (e.key) {
+      case 'ArrowLeft': {
+        e.preventDefault()
+        this.playerSkipBackwards()
+        break
+      }
+      case 'ArrowRight': {
+        e.preventDefault()
+        this.playerSkipForward()
+        break
+      }
+      case ' ': {
+        e.preventDefault()
+        this.playPausePlayer()
+        break
+      }
+      case 'ArrowUp': {
+        e.preventDefault()
+        this.increasePlayerVolume()
+        break
+      }
+      case 'ArrowDown': {
+        e.preventDefault()
+        this.decreasePlayerVolume()
+        break
+      }
+      default:
+    }
+  }
+
+  increasePlayerVolume = () => {
+    const currentVolume = this.video.volume()
+    const newVolume = Math.min(currentVolume + 0.05, 1)
```

Comment:
could we pull the `0.05` out into a constant instead of having magic numbers in the code?  (here and `decreasePlayerVolume`